RoddyCore/src/de/dkfz/roddy/Roddy.java

```diff
@@ -547,8 +544,8 @@ public static final boolean initializeServices(RoddyStartupModes mode) {
      * TODO Does this work from Windows?
      * This uses the proxy vole library.
      */
-    private static void initializeProxySettings() {
-        boolean useProxyForInternetConnection = Boolean.parseBoolean(Roddy.getOrSetApplicationProperty(Constants.APP_PROPERTY_NET_USEPROXY, FALSE));
+    private static void initializeProxySettings() throws ConfigurationError {
```

Comment:
We could try to differentiate between ConfigurationErrors, which come at a later point of execution, and InitiliazationError, which would be here. What do you think?